xtext-website/_posts/releasenotes/2020-06-02-version-2-22-0.md

```diff
@@ -22,11 +22,14 @@ The [Xtext community website](https://www.eclipse.org/Xtext/community.html) and
 
 ## Removals
 
+* Xtext no longer has a direct dependency to ICU4J. If you use ICU4J in your codebase you should also consider to get rid of this dependencies as it will no longer be supported by Eclipse (Platform).
 * As announced in the 2.21 Release the deprecated API in the LSP area was removed.
 * We also removed the outdated leftovers of IDEA Support in `org.eclipse.xtext.xtext.generator`.
 
 ## Deprecations
 
+* `IXtext2EcorePostProcessor` & `XtendXtext2EcorePostProcessor` & `ProjectAwareXtendXtext2EcorePostProcessor` were deprecated as they depend on old Xpand/Xtend(1). As already announced in the [2.21 release notes](2020-03-03-version-2-21-0.md) you should avoid depending on the old Xpand based `org.eclipse.xtext.generator` as well as on Xtend(1) based postprocessing. So if you have an `.ext` file right next to your `.xtext` file or subclass the interface you should take action.
```

Comment:
s/you should take action/it's about time to switch to a manually maintained Ecore model/